src/Chapter2.hs

```diff
@@ -740,7 +769,12 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains p [] = []
+contains p (x:xs)
+  | p `elem` x = x : contains p xs
+  | otherwise = contains p xs
```

Comment:
Absolutely correct implementation! :+1: 

Additionally, you can use the `filter` function instead of manual pattern matching and recursive. `filter` takes a function `(a -> Bool)`, a list of a's (`[a]`) and returns the list where all elements satisfy the given boolean function :slightly_smiling_face: